### PR TITLE
added STOMP related planning files

### PR DESCRIPTION
**Pull request by raghavendersahdev**
This pull request adds the following files for running the STOMP motion planner:
- stomp_planning.yaml
- stomp_planning_pipeline.launch.xml

### CODE REVIEW

File: config/chomp_planning.yaml [hunk 1]
```diff
@@ -13,13 +13,14 @@ hmc_discretization: 0.01
...
 pseudo_inverse_ridge_factor: 1e-4
 animate_endeffector: false
 animate_endeffector_segment: "panda_rightfinger"
 joint_update_limit: 0.1
-collision_clearence: 0.2
+collision_clearence: 0.02
 collision_threshold: 0.07
 random_jump_amount: 1.0
 use_stochastic_descent: true
+trajectory_interpolation_method: "linear"
```
Comment (davetcoleman) [inline config/chomp_planning.yaml]: this PR should only focus on STOMP per title, why mix these CHOMP changes in?

File: config/stomp_planning.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
+stomp/manipulator_rail:
```
Comment (davetcoleman) [inline config/stomp_planning.yaml]: whats a "rail"?

File: config/stomp_planning.yaml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  group_name: panda_arm
+  optimization:
+    num_timesteps: 60
+    num_iterations: 40
+    num_iterations_after_valid: 0    
+    num_rollouts: 30
+    max_rollouts: 30 
+    initialization_method: 1 #[1 : LINEAR_INTERPOLATION, 2 : CUBIC_POLYNOMIAL, 3 : MININUM_CONTROL_COST
```
Comment (davetcoleman) [inline config/stomp_planning.yaml]: remove the ``[`` in comment ?

File: launch/demo_chomp.launch [hunk 4]
```diff
@@ -46,7 +46,6 @@
 
   <!-- Run Rviz and load the default config to see the state of the move_group node -->
   <include file="$(find panda_moveit_config)/launch/moveit_rviz.launch">
-    <arg name="config" value="true"/>
```
Comment (davetcoleman) [inline launch/demo_chomp.launch]: please keep this PR to stomp only (remove this change)

File: config/chomp_planning.yaml (hunk 1 above)
Comment (raghavendersahdev) [reply config/chomp_planning.yaml]: removed this CHOMP change, this was for the additional functionality I added for initialization in CHOMP for [PR#960](https://github.com/ros-planning/moveit/pull/960)

File: config/stomp_planning.yaml (hunk 2 above)
Comment (raghavendersahdev) [reply config/stomp_planning.yaml]: I forgot to change this, sorry, my bad! corrected to `stomp/panda_arm`

File: config/stomp_planning.yaml (hunk 3 above)
Comment (raghavendersahdev) [reply config/stomp_planning.yaml]: done, I completed the open square bracket by adding `]` at the end

File: launch/demo_chomp.launch (hunk 4 above)
Comment (raghavendersahdev) [reply launch/demo_chomp.launch]: done removed this change

File: config/chomp_planning.yaml [hunk 5]
```diff
@@ -3,7 +3,7 @@ max_iterations: 200
 max_iterations_after_collision_free: 5
```
Comment (davetcoleman) [inline config/chomp_planning.yaml]: this is still a CHOMP change, remove

Comment (raghavendersahdev) [reply config/chomp_planning.yaml]: removed all chomp changes

**Comment by raghavendersahdev** [comment]
Sure @davetcoleman , I will also add these files as moveit setup assistant files

**Comment by MohmadAyman** [comment]
This looks great!

@raghavendersahdev  I can add these file to the setup assistant if you don't mind.

**Comment by raghavendersahdev** [comment]
Hi @MohmadAyman  I already added these files to the setup assistant in [this commit](https://github.com/ros-planning/moveit/pull/965/commits/ded331fc5d4b62d172fb77599ba80cf16e7495b9) in [my PR here](https://github.com/ros-planning/moveit/pull/965). Will let you know once the PR gets merged.

**Pull request merged**